src/ngx_http_lua_clfactory.c

```diff
@@ -75,6 +78,7 @@
 #define    LUA_BIG_ENDIAN_8BYTES_CODE_LEN           (8 + 8 + 8)
 #define    LUAC_HEADERSIZE         12
 #define    LUAC_VERSION            0x51
+#endif
```

Comment:
Ditto. Better add a comment to make it clear what condition is this `#endif` belonged to.